src/pytest_reportlog/plugin.py

```diff
@@ -80,6 +86,21 @@ def pytest_sessionfinish(self, exitstatus):
         self._write_json_data(data)
 
     def pytest_runtest_logreport(self, report):
+        if (
+            self._config.option.report_log_exclude_logs_on_passed_tests
+            and report.outcome == "passed"
+        ):
+            report.sections = [
+                s
+                for s in report.sections
+                if s[0]
+                not in [
+                    "Captured log setup",
+                    "Captured log call",
+                    "Captured log teardown",
+                ]
+            ]
```

Comment:
Here you are changing the report log object itself, which might affect other plugins.

Ideally you should remove this information from the `data` dict, as it is owned by this plugin so there's no risk of inadvertently affecting others.

Reply:
yeah, sorry - this wasn't very clever on my part. I have it fixed locally but will add a test and a changelog entry before pushing